cpp/src/parquet/metadata.cc

```diff
@@ -393,6 +393,25 @@ class FileMetaData::FileMetaDataImpl {
     }
   }
 
+  format::RowGroup& get_row_group(int i) {
+    if (!(i < num_row_groups())) {
+      std::stringstream ss;
+      ss << "The file only has " << num_row_groups()
+         << " row groups, requested metadata for row group: " << i;
+      throw ParquetException(ss.str());
+    }
```

Comment:
I don't know that boundschecking is necessary, you can just use `DCHECK_LT`. Can you call this function `row_group(i)`?

Reply:
Sorry - Can you clarify what `row_group` funtion you are refering to?

Reply:
Instead of `get_row_group`